discord.sh

```diff
@@ -5,7 +5,15 @@ socat $SOCAT_ARGS \
     UNIX-CONNECT:$XDG_RUNTIME_DIR/discord-ipc-0 \
     &
 socat_pid=$!
+
+FLAGS='--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --enable-features=UseSkiaRenderer,CanvasOopRasterization'
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -f /dev/nvidia0 ]
```

Comment:
`-f` checks for normal files, while `/dev/nvidia0` is a character device.
```suggestion
if [[ $XDG_SESSION_TYPE == "wayland" ]] && [ -c /dev/nvidia0 ]
```